### PR TITLE
[show] Moving some py imports inside specific cmds

### PR DESCRIPTION
**Pull request by Sabareesh-Kumar-Anandan**
Signed-off-by: Sabareesh Kumar Anandan <sanandan@marvell.com>

**- What I did**
Some py imports in show/main.py script are taking more time on arm cpu.
modules - pkg_resources, netaddr are taking around 500ms causing delay for all show cmds.
These modules are used by only specific cmds. So, I have included these imports only for the commands that are using them.

**- How I did it**

**- How to verify it**
Test all show commands. Execution time of show commands is better than before.

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def get_routing_stack():
 
 
 # Global Routing-Stack variable
-routing_stack = get_routing_stack()
+routing_stack = "frr"
```
Comment (lguohan) [inline show/main.py:59]: need to think about this.

File: show/main.py [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+#import netaddr
 import os
 import subprocess
 import sys
 
 import click
 from natsort import natsorted
 import netifaces
-from pkg_resources import parse_version
+#from pkg_resources import parse_version
```
Comment (jleveque) [inline show/main.py:12]: Why comment out?

File: show/main.py [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 
 import feature
 import interfaces
 import kube
-import mlnx
+#import mlnx
```
Comment (jleveque) [inline show/main.py:17]: I believe commenting this out completely will break Mellanox platforms.

File: show/main.py (hunk 2 above)
Comment (Sabareesh-Kumar-Anandan) [reply show/main.py:12]: This module import is taking more time(~300ms) on arm cpu. As this is required only for "show ntp" cmd, I have added it inside ntp command. see below

File: show/main.py (hunk 3 above)
Comment (Sabareesh-Kumar-Anandan) [reply show/main.py:17]: this is included after checking for mellanox platform. see below (line 1008)

File: show/main.py (hunk 1 above)
Comment (lguohan) [reply show/main.py:59]: this reduce the function, can you exclude this in this pr? also it does not fit in the pr title.

File: utilities_common/cli.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 import click
 import json
-import netaddr
+#import netaddr
```
Comment (lguohan) [inline utilities_common/cli.py:8]: how much delay is this one? btw, we moved to python3, is this still an issue?

File: show/main.py [hunk 5]
```diff
@@ -1,20 +1,20 @@
 #! /usr/bin/python -u
 
 import json
-import netaddr
+#import netaddr
```
Comment (lguohan) [inline show/main.py:4]: do not comment, please remove.

File: utilities_common/cli.py (hunk 4 above)
Comment (Sabareesh-Kumar-Anandan) [reply utilities_common/cli.py:8]: takes around 200ms

File: show/main.py (hunk 1 above)
Comment (Sabareesh-Kumar-Anandan) [reply show/main.py]: removed it. will think of better way of doing this

File: show/main.py (hunk 5 above)
Comment (Sabareesh-Kumar-Anandan) [reply show/main.py]: done

**Comment by Sabareesh-Kumar-Anandan** [comment]
Hi @lguohan, 
I see 3 tests have failed. I don't think failure is caused by this change. Is it possible to retrigger the tests?

**Comment by lguohan** [comment]
retest this please

**Review by lguohan — APPROVED** [review]

**Comment by lguohan** [comment]
retest this please

**Pull request merged**